addons/hr/static/tests/helpers/mock_models.js

```diff
@@ -1,7 +1,7 @@
 odoo.define('hr/static/tests/helpers/mock_models.js', function (require) {
 'use strict';
 
-const MockModels = require('mail/static/tests/helpers/mock_models.js');
+const MockModels = require('@mail/../tests/helpers/mock_models')[Symbol.for("default")];
```

Comment:
I would move the `tests` folder into `src` to avoid this confusing `..`
(I think it was a bad idea from them to remove `static/src` from the name.)

Reply:
As discussed, apparently this naming for tests is on purpose, so ok to keep it like this.